cfg/Locations.json

```diff
@@ -16,6 +16,7 @@
   "Central Hub": {
     "East": "Bar",
     "West": "Medical Bay",
+    "South": "Landing Dock",
```

Comment:
Great catch

Reply:
Yeah I noticed it when I was controlling the character and I couldn't move south